### ISSUE TITLE
Feature Request: ownCloud logfile - line breakes before n#<line number>

### ISSUE THREAD

**Issue by mmattel** [icewind1991/logreader#14]
Hello @icewind1991 

referencing https://github.com/owncloud/core/issues/24984
Readability would improve a lot. Would be great to see that!!

**Comment by icewind1991** [comment]
can you send me the part of the logfile with the background job exception in it

**Comment by mmattel** [comment]
Here it is. I neutralized some data. Thanks for taking care of :smile:

```
{"reqId":"3Z2LbWILbBUS2L0zWGki","remoteAddr":"0.0.0.0","app":"core","message":"Error while running background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":2:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:200:\"\/a_file\";s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:8:\"a_user\";}): {\"Exception\":\"OC\\\\User\\\\NoUserException\",\"Message\":\"Attempted to initialize mount points for null user and no user in session\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/lib\\\/storage.php(102): OC\\\\Files\\\\Filesystem::initMountPoints(false)\\n#1 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/lib\\\/storage.php(682): OCA\\\\Files_Versions\\\\Storage::getUidAndFilename('\\\/a_file...')\\n#2 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php(61): OCA\\\\Files_Versions\\\\Storage::expire('\\\/a_file...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(34): OCA\\\\Files_Versions\\\\Command\\\\Expire->handle()\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(171): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/files\\\/filesystem.php\",\"Line\":376}","level":3,"time":"2016-06-06T14:24:42+02:00","method":"GET","url":"\/cron.php","user":"--"}
{"reqId":"Q378iyANzmN1bK3O10jR","remoteAddr":"0.0.0.0","app":"core","message":"Login failed: 'system' (Remote IP: '0.0.0.0')","level":2,"time":"2016-06-06T14:30:03+02:00","method":"REPORT","url":"\/remote.php\/dav\/addressbooks\/system\/system\/system","user":"--"}
{"reqId":"pNuChVqByXHGo+31tvuH","remoteAddr":"0.0.0.0","app":"core","message":"Error while running background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":2:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:200:\"\/a_file\";s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:8:\"a_user\";}): {\"Exception\":\"OC\\\\User\\\\NoUserException\",\"Message\":\"Attempted to initialize mount points for null user and no user in session\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/lib\\\/storage.php(102): OC\\\\Files\\\\Filesystem::initMountPoints(false)\\n#1 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/lib\\\/storage.php(682): OCA\\\\Files_Versions\\\\Storage::getUidAndFilename('\\\/a_file...')\\n#2 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php(61): OCA\\\\Files_Versions\\\\Storage::expire('\\\/a_file...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(34): OCA\\\\Files_Versions\\\\Command\\\\Expire->handle()\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(171): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/files\\\/filesystem.php\",\"Line\":376}","level":3,"time":"2016-06-06T14:30:19+02:00","method":"GET","url":"\/cron.php","user":"--"}
```

**Comment by icewind1991** [comment]
https://github.com/icewind1991/logreader/commit/d80ac71c49301c4fa398f9e34a9ab68ae6bdf785 adds fancy parsing of exceptions in background jobs

![exception in background job](https://i.imgur.com/72DdOkX.png)

**Comment by mmattel** [comment]
cool :+1: 
I did not find this in the merged/closed PR´s when I was looking at it yesterday, what a coincidence :smile: 
Mind to do a downloadable prebuild as this also would then include 9.1 readiness?

**Comment by icewind1991** [comment]
> Mind to do a downloadable prebuild as this also would then include 9.1 readiness?

https://icewind.me/owncloud/index.php/s/hkUnqCjg1fRo81j

**Comment by mmattel** [comment]
Many thanks for your support!
Feedback:
I uninstalled the old version 1.1.2 of logreader
Then I successfully installed logreader 1.1.3 and enabled it.
When clicking on the logreader symbol, it changes to the screen but nothing happens.
There is nothing in the owncloud nor the nginx error log.
The logreader and the user buttons are dead, I have to go back with the browser.
My instance is running: ownCloud 9.0.2 RC1 (daily) Build:2016-04-28T03:03:12+00:00 795f321a1985b682bff584f252dd6f2fd95dad65
![image](https://cloud.githubusercontent.com/assets/3321281/15864235/92eb1ff4-2cd6-11e6-8af2-840a924e5c60.png)
